test/fixtures/rules/std_rule_cases/L057.yml

```diff
@@ -0,0 +1,81 @@
+rule: L057
+
+test_fail_special_chars_create_table_space:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Internal Space" INT
+    )
+
+test_fail_special_chars_create_table_gt:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Greater>Than" INT
+    )
+
+test_fail_special_chars_create_table_lt:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Less<Than" INT
+    )
+
+test_fail_special_chars_create_table_hash_unquoted:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        Number# INT
+    )
+  configs:
+    core:
+      dialect: tsql
+
+test_pass_special_chars_create_table_none:
+  pass_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "No_Internal_Space" INT,
+        GreaterThan INT,
+        LessThan INT,
+        NumberVal INT
+    )
+
+test_pass_special_chars_create_table_none_quoted_tsql:
+  pass_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        [No_Internal_Space] INT,
+        [GreaterThan] INT,
+        [LessThan] INT,
+        NumberVal INT
```

Comment:
is it possible to have quotes and square brackets or are the two mutually exclusive?
Not a t-sql user myself so curious 😄

Reply:
The two are mostly interchangeable.  You wouldn't use both, but one or the other.